.gitignore

```diff
@@ -9,6 +9,7 @@ tmp/sass-cache
 tmp/binlog
 tmp/rspec_guard_result
 tmp/capybara
+tmp/letter_opener
```

Comment:
I was using the letter opener gem for other issues, indeed this is irrelevant to keep it here